src/helpers/Query.ts

```
@@ -77,25 +81,27 @@ export class Query {
 
-			table_name = options.schema.table
+			table_name = (options as any).schema.table
 		}
 
 		try {
 			let result
 
 			switch (action) {
 				case QueryPerform.CREATE:
+
 					const createOptions = options as DatabaseCreateOneOptions
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)